tests/parsers/test_navdata.py

```diff
@@ -1284,19 +1290,123 @@ def test_add_pandas_df(df_simple, add_df):
         pd.DataFrame to add to NavData
     """
     data = NavData(pandas_df=df_simple)
-    data.add(pandas_df=add_df)
+    data.concat(NavData(pandas_df=add_df),axis=1,inplace=True)
     new_df = data.pandas_df()
     add_row_num = add_df.shape[0]
     subset_df = new_df.iloc[-add_row_num:, :].reset_index(drop=True)
-    pd.testing.assert_frame_equal(subset_df, add_df,
+    pd.testing.assert_frame_equal(subset_df.sort_index(axis=1),
+                                  add_df.sort_index(axis=1),
                                   check_index_type=False)
 
     # test adding to empty NavData
     data_empty = NavData()
-    data_empty.add(pandas_df=add_df)
-    pd.testing.assert_frame_equal(add_df, data_empty.pandas_df(),
+    data_empty.concat(NavData(pandas_df=add_df),axis=1,inplace=True)
+    pd.testing.assert_frame_equal(add_df.sort_index(axis=1),
+                                  data_empty.pandas_df().sort_index(axis=1),
                                   check_index_type=False)
 
+def test_concat(df_simple):
+    """Test concat functionaltiy.
+
+    Parameters
+    ----------
+    df_simple : pd.DataFrame
+        Simple pd.DataFrame with which to initialize NavData.
+
+    """
+
+    navdata_1 = NavData(pandas_df=df_simple)
+    navdata_2 = navdata_1.copy()
```

Comment:
maybe you can also add tests on what happens if an completely empty navdata class is given as input? the concatenated one should be same as input one, while still maintaining the navdata class instance.

Reply:
Added suggested test with empty Navdata. The test should fail if `axis=0` because the number of columns must be the same.